test/test_examples_2d_curved.jl

```diff
@@ -174,6 +174,13 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
       tspan = (0.0, 0.3))
   end
 
+  @testset "elixir_hypdiff_nonperiodic_curved.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_hypdiff_nonperiodic_curved.jl"),
+      l2   = [0.8799784202842884, 0.8538496716674346, 0.7861306028768382],
+      linf = [1.0772194937514343, 1.9152049495265542, 2.1515261806766617],
+      tspan = (0.0, 0.1))
+  end
```

Comment:
How long does it take to run this until convergence? Did you play with the CFL number?

Reply:
It takes quite awhile (until about `t_end = 29`. I played with the CFL a little bit, but I use `polydeg = 6` for this test and the special hypdiff time integrator was optimized for `polydeg = 3`. So I was not sure how much leeway I had with the CFL